roles/network_plugin/multus/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 multus_conf_file: "auto"
 multus_cni_conf_dir_host: "/etc/cni/net.d"
-multus_cni_bin_dir_host: "{{ '/usr/libexec/cni' if container_manager == 'crio' else '/opt/cni/bin' }}"
```

Comment:
@crandles Could you take a look at this change?
This part came from https://github.com/kubernetes-sigs/kubespray/pull/5701 and this reverts it.

Reply:
I was basing my changes off of this: https://github.com/intel/multus-cni/blob/c85b79f5ff5bcacaa45e2135d29e9afb6b84ed9b/images/multus-daemonset-crio.yml#L166-L198 but agree it doesn't look necessary at this time

Reply:
Oh, didn't think that was a recent change, but not sure it is ndeeded but you tell me (well as you approved the changes I guess you're fine with it)